www/index.md

```diff
@@ -0,0 +1,26 @@
+# Compensated
+Build products. Get paid.
+
+
+### Expand Your Sales Channels
+You started with a website, and now you have an iOS app.
+And oh yea, there’s that old Gumroad storefront you were experimenting with that has some recurring payments set up.
+
+No matter how you get (or got!) paid, Compensated standardizes the different payment processers data structures and event types so you can add sales channels without losing momentum on your customer facing features.
+
+### Prevent Vendor Lock-In
+Tired of your current payment providers fee structure?
+Sick of the weird refund policy that locks your account?
+
+Compensated helps you safely transition your customers to new payment processors without requiring system rewrites.
+
+### Read the Source, Luke
+All our source code is publicly available.
+We build in the open, and grant usage rights freely to any and all individuals. 
```

Comment:
...Except the individuals described below

Reply:
I'm not sure what you mean here?

Reply:
"We ... grant usage rights freely to any and all individuals." But then we say "we have legal standing to refuse to grant usage rights to organizations that violate human rights". (Which is cool.) So it's not to "any and all" individuals. And "freely" is a bit confusing since it's a paid product, though technically used in a different sense here. I guess I just don't get what this part is trying to communicate.

Reply:
Ah, so the thing to keep in mind is that an individual acting on behalf of an organization is distinct from an individual acting on behalf of oneself. Someone who say, installs the gem and pushes it to production for a service being operated by the red cross, is an individual acting on behalf of an organization. That person _may_ choose to claim that the software they are using the gem in is operated by them as an individual, but it would be unlikely to hold water.

I am definitely interested in hearing how you would phrase this to disambiguate between the two things; but it's not a paradox; I swear!

Reply:
Ah I see... So we don't ever want to prevent an _individual_ from using, even a really bad (for some definition we determine) individual? And is a sole proprietor still also an individual?